src/components/play/baseComponents/YoutubeLink.tsx

```diff
@@ -24,23 +24,22 @@ const YoutubeLink: React.FC<YoutubeLinkProps> = (props) => {
     }
   };
 
-  if (isPhone()) {
-    opts = {
-      playerVars: {
-        playsinline: 0,
-        fs: 0
-      },
-    };
+  const rememberYoutube = () => {
+    SetYoutubeClick();
+    setTimeout(() => {
+      UnsetYoutubeClick();
+    }, 2000);
```

Comment:
@scholar6admin youtube rotation delay 2sec. in browser stack it takes around 2sec for orientation event to fire. during those 2sec if user rotate phone it will allow no warning will be shown.